wzdx/raw_to_standard/planned_events.py

```diff
@@ -683,14 +683,32 @@ def get_cross_streets_from_description(description: str) -> tuple[str, str]:
         return ("", "")
 
 
+def get_mileposts_from_description(message: str) -> tuple[str, str]:
```

Comment:
This is a really interesting function, let's add some unit tests to show capabilities and limitations so we know what to expect when the throw us a curve ball down the line

Reply:
Also, is this just making the assumption that the Exit number will be the same as milepost?

Reply:
The description text is currently automatically generated, so for now, it parses well. Good point that I was processing the exit numbers, I switched this to read the Mile Post measurements. I also implemented the following priorities for determine mileposts:
1. geospatial route details, if start and end routes match
2. planned event properties (startMarker and endMarker)
3. description parsing